packages/slate/src/models/state.js

```diff
@@ -21,6 +21,7 @@ const DEFAULTS = {
   selection: Selection.create(),
   history: History.create(),
   data: new Map(),
+  isNative: false,
```

Comment:
Curious if this should go as a flag on the change object instead along with merge and save